_mocks_/fileMock.js

```diff
@@ -0,0 +1 @@
+module.exports = "test-file-stub";
```

Comment:
curious what this does?

Reply:
its for jest, because i import .png file inside the .tsx file it cant detect the module so need to add some things inside to make it work. this is just part of the things needed for it to work